{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"pytest_version": "7.4.4", "$report_type": "SessionStart"}
{"nodeid": "tests18_7_5.py::test_negate_of_divisors_large_list", "location": ["tests18_7_5.py", 30, "test_negate_of_divisors_large_list"], "keywords": {"test_negate_of_divisors_large_list": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00015213299775496125, "start": 1729628665.295008, "stop": 1729628665.2951605, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw1", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw1>"}
{"nodeid": "tests18_7_5.py::test_list_of_zeros", "location": ["tests18_7_5.py", 3, "test_list_of_zeros"], "keywords": {"test_list_of_zeros": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0002643610059749335, "start": 1729628665.2951248, "stop": 1729628665.2953897, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw0>"}
{"nodeid": "tests18_7_5.py::test_negate_of_lists", "location": ["tests18_7_5.py", 17, "test_negate_of_lists"], "keywords": {"test_negate_of_lists": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00018631700368132442, "start": 1729628665.29893, "stop": 1729628665.299117, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw2>"}
{"nodeid": "tests18_7_5.py::test_list_of_zeros", "location": ["tests18_7_5.py", 3, "test_list_of_zeros"], "keywords": {"test_list_of_zeros": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_zeros():", "        list_of_zeros = [0 for i in range(0, 3 * (584 + 1))]", ">       assert sum_ints_div_by_either_nums(list_of_zeros) == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/tests18_7_5.py", "lineno": 6, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_ints_div_by_either_nums(nums):", "        for i in range(543, 585):", "            if nums[i] % -553 == 0 or nums[i] % -737 == 0:", ">               answer += nums[i]", "E               UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"], "reprfuncargs": {"args": [["nums", "[0, 0, 0, 0, 0, 0, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_zeros():", "        list_of_zeros = [0 for i in range(0, 3 * (584 + 1))]", ">       assert sum_ints_div_by_either_nums(list_of_zeros) == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/tests18_7_5.py", "lineno": 6, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_ints_div_by_either_nums(nums):", "        for i in range(543, 585):", "            if nums[i] % -553 == 0 or nums[i] % -737 == 0:", ">               answer += nums[i]", "E               UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"], "reprfuncargs": {"args": [["nums", "[0, 0, 0, 0, 0, 0, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00021987699437886477, "start": 1729628665.2956657, "stop": 1729628665.2958863, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw0>"}
{"nodeid": "tests18_7_5.py::test_list_of_zeros", "location": ["tests18_7_5.py", 3, "test_list_of_zeros"], "keywords": {"test_list_of_zeros": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00016816399875096977, "start": 1729628665.31287, "stop": 1729628665.313039, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw0>"}
{"nodeid": "tests18_7_5.py::test_negate_of_lists", "location": ["tests18_7_5.py", 17, "test_negate_of_lists"], "keywords": {"test_negate_of_lists": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_negate_of_lists():", "        initial_list = [i for i in range(0, 584 + 1)]", "        neg_initial_list = [-i for i in range(0, 584 + 1)]", ">       assert sum_ints_div_by_either_nums(initial_list) == -(sum_ints_div_by_either_nums(neg_initial_list))"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/tests18_7_5.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_ints_div_by_either_nums(nums):", "        for i in range(543, 585):", "            if nums[i] % -553 == 0 or nums[i] % -737 == 0:", ">               answer += nums[i]", "E               UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_negate_of_lists():", "        initial_list = [i for i in range(0, 584 + 1)]", "        neg_initial_list = [-i for i in range(0, 584 + 1)]", ">       assert sum_ints_div_by_either_nums(initial_list) == -(sum_ints_div_by_either_nums(neg_initial_list))"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/tests18_7_5.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_ints_div_by_either_nums(nums):", "        for i in range(543, 585):", "            if nums[i] % -553 == 0 or nums[i] % -737 == 0:", ">               answer += nums[i]", "E               UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00022194300254341215, "start": 1729628665.299423, "stop": 1729628665.2996454, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw2>"}
{"nodeid": "tests18_7_5.py::test_negate_of_lists", "location": ["tests18_7_5.py", 17, "test_negate_of_lists"], "keywords": {"test_negate_of_lists": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00014251899847295135, "start": 1729628665.3129451, "stop": 1729628665.3130882, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw2", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw2>"}
{"nodeid": "tests18_7_5.py::test_negate_of_divisors_large_list", "location": ["tests18_7_5.py", 30, "test_negate_of_divisors_large_list"], "keywords": {"test_negate_of_divisors_large_list": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_negate_of_divisors_large_list():", "        neg_div1 = --553", "        neg_div2 = --737", "        mul_of_negs = neg_div1 * neg_div2", "        mul_list = [mul_of_negs for _ in range(0, (584 + 1) * 100)]", "        expected_sum = sum(mul_list[543:584 + 1])", ">       assert sum_ints_div_by_either_nums(mul_list) == expected_sum"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/tests18_7_5.py", "lineno": 37, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_ints_div_by_either_nums(nums):", "        for i in range(543, 585):", "            if nums[i] % -553 == 0 or nums[i] % -737 == 0:", ">               answer += nums[i]", "E               UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"], "reprfuncargs": {"args": [["nums", "[407561, 407561, 407561, 407561, 407561, 407561, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_negate_of_divisors_large_list():", "        neg_div1 = --553", "        neg_div2 = --737", "        mul_of_negs = neg_div1 * neg_div2", "        mul_list = [mul_of_negs for _ in range(0, (584 + 1) * 100)]", "        expected_sum = sum(mul_list[543:584 + 1])", ">       assert sum_ints_div_by_either_nums(mul_list) == expected_sum"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/tests18_7_5.py", "lineno": 37, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def sum_ints_div_by_either_nums(nums):", "        for i in range(543, 585):", "            if nums[i] % -553 == 0 or nums[i] % -737 == 0:", ">               answer += nums[i]", "E               UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"], "reprfuncargs": {"args": [["nums", "[407561, 407561, 407561, 407561, 407561, 407561, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_D/Q18/codegemma_results_5/Folder_7/generated_answer.py", "lineno": 4, "message": "UnboundLocalError: cannot access local variable 'answer' where it is not associated with a value"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0016288479964714497, "start": 1729628665.2954206, "stop": 1729628665.29705, "extras": [], "$report_type": "TestReport", "item_index": 4, "worker_id": "gw1", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw1>"}
{"nodeid": "tests18_7_5.py::test_negate_of_divisors_large_list", "location": ["tests18_7_5.py", 30, "test_negate_of_divisors_large_list"], "keywords": {"test_negate_of_divisors_large_list": 1, "tests18_7_5.py": 1, "Folder_7": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00014534099318552762, "start": 1729628665.3140335, "stop": 1729628665.3141794, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw1", "testrun_uid": "15e410cdaa6840ae9639fe7cea117e18", "node": "<WorkerController gw1>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
